### PR TITLE
feat: add jsd loss and asymmetric loss

### PR DESCRIPTION
**Pull request by wcrzlh**
Thank you for your contribution to the MindCV repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests

## Motivation

Jsd loss and asymmetric loss are added.

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

Related issue: https://github.com/mindspore-lab/mindcv/issues/621

### CODE REVIEW

File: mindcv/loss/asymmetric.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    def construct(self, logits, labels):
+        x_sigmoid = ops.Sigmoid()(logits)
+        xs_pos = x_sigmoid
+        xs_neg = 1 - x_sigmoid
+
+        if self.clip > 0:
+            xs_neg = ops.clip_by_value(xs_neg + self.clip, clip_value_max=Tensor(1.0))
+
+        los_pos = labels * ops.Log()(ops.clip_by_value(xs_pos, clip_value_min=Tensor(self.eps)))
```
Comment (geniuspatrick) [inline mindcv/loss/asymmetric.py:23]: 这些算子统一看下有没有函数式接口的，有就替换下

**Review by geniuspatrick — CHANGES_REQUESTED** [review]

File: mindcv/loss/asymmetric.py (hunk 1 above)
Comment (vigo999) [reply mindcv/loss/asymmetric.py]: @wcrzlh finished???

Comment (wcrzlh) [reply mindcv/loss/asymmetric.py]: finished

**Comment by geniuspatrick** [comment]
please add some unit tests

**Comment by wcrzlh** [comment]
added

**Review by SamitHuang — APPROVED** [review]

**Review by geniuspatrick — APPROVED** [review]
